src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
We already have a dependency on cmake because of this. 

The biggest thing we would get is a consistent build between the platforms. I don't know if there are any issues with between the differences between make and msbuild generates stuff but in general I'm trying to cut down on the differences between our different build environments. Do we get any benefits to having it generate msbuild files? The only one I can see is being able to load these projects in VS but most of these are very small native libraries so that benefit seems minimal to me.